src/client/applications/turnilo-application/turnilo-application.tsx

```diff
@@ -34,18 +34,17 @@ import { replaceHash } from "../../utils/url/url";
 import { CubeView } from "../../views/cube-view/cube-view";
 import { HomeView } from "../../views/home-view/home-view";
 import { NoDataView } from "../../views/no-data-view/no-data-view";
-import "./swiv-application.scss";
+import "./turnilo-application.scss";
 
-export interface SwivApplicationProps {
+export interface TurniloApplicationProps {
   version: string;
   user?: User;
```

Comment:
Perhaps is not needed but I'm not fully sure. Is it possible to set user from requestdecorator https://github.com/allegro/turnilo/blob/master/docs/security.md ?

Reply:
But for this case it shouldn't be done on server side? Right now `user` is not really used.